packages/cbioportal-clinical-timeline/src/TimelineStore.ts

```diff
@@ -14,6 +14,7 @@ import {
 import _ from 'lodash';
 import $ from 'jquery';
 import autobind from 'autobind-decorator';
+import { ClinicalEvent } from 'cbioportal-ts-api-client';
```

Comment:
we should probably try to keep the timeline library free of reference to other cbio libraries. lets discuss

Reply:
fixed